assets/js/youcan-pay.js

```diff
@@ -22,7 +22,7 @@ jQuery(function ($) {
 
     function detach_loader($form, loader) {
         let target = '.woocommerce';
-        if (gateways.cash_plus === parseInt(window.ycPay.selectedGateway)) {
+        if ((window.ycPay != null) && (gateways.cash_plus === parseInt(window.ycPay.selectedGateway))) {
```

Comment:
`!==` or `typeof window.ycPay !== 'undefined'`

Reply:
https://github.com/NextmediaMa/youcan-pay-for-woocommerce/pull/33/commits/9c7cd71ecca0724bb9764f45223c41b5b4ffffb3